app.js

```diff
@@ -176,6 +180,111 @@ function resetScheduler() {
   });
 }
 
+const serviceAccountKeyFile = "./article-bot-database-2c885470e1c3.json";
```

Comment:
what is this json file?

Reply:
this json contains info regarding google sheet api's

Reply:
change the name of this file